### ISSUE TITLE
Catch exception when pools are empty in python 3 MMF example

### ISSUE THREAD

**Issue by joeholley** [GoogleCloudPlatform/open-match#49]
```
Retrieving pool 'defaultPool'.
'defaultPool': count 000000 | elapsed 0.002
Retrieving pool 'supportPool'.
'supportPool': count 000000 | elapsed 0.002
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/random.py", line 262, in choice
    i = self._randbelow(len(seq))
  File "/usr/local/lib/python3.5/random.py", line 239, in _randbelow
    r = getrandbits(k)          # 0 <= r < 2**k
ValueError: number of bits must be greater than zero

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./harness.py", line 80, in <module>
    results = mmf.makeMatches(profile_dict, player_pools)
  File "/usr/src/open-match/examples/functions/python3/mmlogic-simple/mmf.py", line 34, in makeMatches
    player['id']  = random.choice(list(player_pools[player['pool']]))
  File "/usr/local/lib/python3.5/random.py", line 264, in choice
    raise IndexError('Cannot choose from an empty sequence')
IndexError: Cannot choose from an empty sequence
```
Should catch and exit cleanly.

**Comment by joeholley** [comment]
This will be included in the 0.4.0 release scheduled for late Feb 2019.

**Comment by yfei1** [comment]
Close this issue since the old python examples are gone.
